docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,240 @@
+---
+title: "Create a Facial Verification System"
+linkTitle: "Verification System"
+weight: 40
+type: "docs"
+description: "Configure a verification system module to implement a facial verification security system on your smart machine."
+image: "/tutorials/verification-system/disarmed.png"
+imageAlt: "Bijan disarming the facial verification system."
+images: ["/tutorials/verification-system/disarmed.png"]
+tags: ["mlmodel", "vision", "services", "security", "canera"]
+authors: []
+languages: []
+viamresources: ["mlmodel", "vision", "camera"]
+level: "Intermediate"
+date: "28 November 2023"
+cost: "0"
+no_list: true
+# SMEs: Bijan Haney
+---
+
+In the [registry](https://app.viam.com/module/viam-labs/verification-system), Viam Labs provides a model of [vision service classifier](/ml/vision/classification/) called `verification-system`.
+Follow this tutorial to learn how to configure this module and set up your own security system to detect, verify, and alarm based on specified detections from nested [vision service detectors](/ml/vision/detection/).
+
+You can use whatever combination of detectors and camera you would like with this module, but follow along with our recommendations to layer an [`mlmodel` person detector](#configure-an-mlmodel-person-detector) and [facial detector](#configure-a-facial-detector) together, creating a security system with facial recognition.
```

Comment:
This currently starts off in a way that only someone familiar with the product would understand what is being built. 

Let's try to make this a bit more like a sample project someone built not to show off the module but because they wanted to build the project. Let's start by spelling out what it is we're building and what the core functionality is (how it can recognize a face and that disarms the system etc). Being more specific with this helps the reader understand what's being built. We also should get a picture of the built project - you can spend some time building it. Or maybe take a picture of Bijan's. Cardboard and tape is fine to hold things together if need be. But let's help the audience visualise what is being built. The video of the disarming would also help explain the functionality.